docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
```

Comment:
Shouldn't these be in a different file so that this file remains untouched? Otherwise when checking out with git, changing these will cause git to have a modified file, and this one shouldn't be committed.